### PR TITLE
Update dependency to latest Rubocop 1.50

### PR DESCRIPTION
**Pull request by JulienItard**
edit : Hello ! I would like to use Rubocop higher than 1.41. I cant because of this :)

______________________________________________________________________

Before submitting the PR make sure the following are checked:

- [x] Feature branch is up-to-date with `main` (if not - rebase it).
- [x] Squashed related commits together.
- [ ] Added tests.
- [ ] Updated documentation.
- [x] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
- [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

File: rubocop-capybara.gemspec [hunk 1]
```diff
@@ -32,5 +32,5 @@ Gem::Specification.new do |spec|
     'rubygems_mfa_required' => 'true'
   }
 
-  spec.add_runtime_dependency 'rubocop', '~> 1.41'
+  spec.add_runtime_dependency 'rubocop', '~> 1.50'
```
Comment (ydah) [inline rubocop-capybara.gemspec:35]: Currently, `NewCops: disable` is set, so the following specification may be sufficient.

```suggestion
  spec.add_runtime_dependency 'rubocop', '>= 1.41', '< 2.0'
```

**Comment by bquorning** [comment]
Why do we need to require a newer version of RuboCop?

File: rubocop-capybara.gemspec (hunk 1 above)
Comment (koic) [reply rubocop-capybara.gemspec:35]: FYI: https://github.com/rubocop/rubocop-rspec/pull/1635#discussion_r1179187224

Comment (ydah) [reply rubocop-capybara.gemspec:35]: Sorry, I had overlooked that the patch version was not specified 😅.

**Comment by JulienItard** [comment]
> Why do we need to require a newer version of RuboCop?

because I would like to use Rubocop greater than 1.41 :)

**Comment by pirj** [comment]
There's nothing that prevents you from doing so, @JulienItard

**Comment by pirj** [comment]
We bump the minimum `rubocop` dependency when we need something new from `rubocop`.
There is nothing that prevents users from using the latest 1.xx `rubocop` with any of `rubocop-capybara` versions.

**Pull request closed without merging**